illumina/illumina.nf

```diff
@@ -1,10 +1,20 @@
 #!/usr/bin/env nextflow
 
-params.INDEX = "gs://prj-int-dev-covid19-nf-gls/prepro/illumina.index.tsv"
-params.STOREDIR = "gs://prj-int-dev-covid19-nf-gls/prepro/storeDir"
-params.OUTDIR = "gs://prj-int-dev-covid19-nf-gls/prepro/results"
+//TSV file of samples to process
+params.INDEX = "gs://sands-nf-tower/illumina5.tsv"
+params.STOREDIR = "gs://sands-nf-tower/storeDir"
+params.OUTDIR = "gs://sands-nf-tower/results"
+params.CONFIG_YAML = "gs://sands-nf-tower/config.yaml"
+
+//If using LSF, uncomment the following variables and provide an appropriate path to the files
```

Comment:
Is it true that you can use GCS when launching LSF jobs from NXF Tower?

Reply:
I think we should have a separate section for LSF and for Google. I'll try and add that.